pkg/sdk/logging/model/output/logdna.go

```diff
@@ -25,7 +25,7 @@ type _hugoLogDNA interface{} //nolint:deadcode,unused
 
 // +kubebuilder:object:generate=true
 // +docName:"[LogDNA Output](https://github.com/logdna/fluent-plugin-logdna)"
-// This plugin has been designed to output logs to LogDNA. Example Deployment: [Transport Nginx Access Logs into LogDNA with Logging Operator](https://raw.githubusercontent.com/banzaicloud/logging-operator/master/docs/examples/logging_output_logdna.yaml)
+// This plugin has been designed to output logs to LogDNA. Example Deployment: [Transport Nginx Access Logs into LogDNA with Logging Operator](https://raw.githubusercontent.com/kube-logging/logging-operator/master/docs/examples/logging_output_logdna.yaml)
```

Comment:
This example does not exist: https://github.com/kube-logging/kube-logging.github.io/tree/master/content/docs/examples